cppcache/src/TcrMessage.cpp

```diff
@@ -2960,6 +3254,236 @@ void TcrMessage::readHashSetForGCVersions(
   }
 }
 
+
+TcrMessageDestroyRegion::~TcrMessageDestroyRegion() {}
```

Comment:
Any not `~X() noexcept = default`?
Why not in the header?

Reply:
We are moving all implementation out of the headers.